src/beyond_types/partial_things.md

```diff
@@ -53,14 +53,14 @@ namespace ExampleNamespace
+While partial classes and methods offer significant advantages in managing large codebases and improving code modularity, they can also introduce challenges if misused. One of the most common misuses of partial classes is scattering related code across multiple files without a clear organizational strategy. This can lead to difficulty in understanding the full behavior of a class because its logic is fragmented. Instead of enhancing clarity, partial classes can contribute to code that is harder to navigate and maintain. To avoid this situation, developers should use partial classes with a clear intent and a consistent structure. Each part of a partial class should encapsulate logically grouped functionality, such as separating auto-generated code from custom logic, rather than splitting code arbitrarily.
 
-Similarly, partial methods, while useful for providing extension points, can lead to confusion if overused or poorly documented. For example, declaring too many partial methods without clear documentation on their intended use can make it difficult for other developers to understand which methods are critical and which are optional. This misuse can lead to a fragmented implementation where the flow of control is obscured, making the codebase harder to maintain and debug. Developers should use partial methods sparingly and ensure that their purpose is well-documented, indicating when and why a method might be implemented or left unimplemented.
+Similarly, partial methods, while useful for providing extension points, can lead to confusion if overused or poorly documented. For example, declaring too many partial methods without clear documentation on their intended use can make it difficult for other developers to understand which methods are critical and which are optional. This misuse can lead to a fragmented implementation where the flow of control is obscured, making the codebase harder to maintain and debug. Developers should use partial methods sparingly and ensure their purpose is well-documented, indicating when and why a method might be implemented or left unimplemented.
 
-#### Common Mistakes and How to Avoid Them**
+## Common mistakes and how to avoid them**<!--TODO: Jeff, do these asterisks mean something?-->
```

Comment:
almost certainly just a typo or failure to convert the formatting from bold to a header